File: PasswordGenerator.vb
module module1
        sub main
                dim posschar As String = "89plokm0ijnuhb7ygv6tfc5rdx4esz3waq12QWERTYUIOPLAKSJDHFGMZNXBCV[]{};:/?.,<>!@#$%^&*()-=_+",myPass=""
                dim achar As New random()
                for i = 1 To 8
                        myPass &= posschar(achar.next(posschar.length()))
                next
                console.write("Your password is :" & vbnewline & myPass)
        end Sub
end Module

File: PasswordGenerator.cs
using System;
public class PasswordGenerator
{
	static void Main()
	{
		string all="89plokm0ijnuhb7ygv6tfc5rdx4esz3waq12QWERTYUIOPLAKSJDHFGMZNXBCV[]{};:/?.,<>!@#$%^&*()-=_+", mypass="";
		var rnd = new Random();
		int pw_len=8;
		for (int i=1; i<=pw_len; i++)
			mypass += all[rnd.Next(all.Length)];
		Console.WriteLine("Your password is:");
		Console.Write(mypass);
	}
}
